gms/impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java

```diff
@@ -198,6 +219,30 @@
     return RestliUtils.toTask(() -> new StringArray(_searchService.getBrowsePaths(urnToEntityName(urn), urn)));
   }
 
+  /*
+  Used to enable writes in GMS after data migration is complete
+   */
+  @Action(name = "delete")
+  @Nonnull
+  public Task<RollbackResponse> deleteEntity(@ActionParam(PARAM_URN) @Nonnull String urnStr) throws URISyntaxException {
+    Urn urn = Urn.createFromString(urnStr);
+
+    return RestliUtils.toTask(() -> {
+      RollbackResponse response = new RollbackResponse();
+
+      EbeanEntityService.RollbackRunResult result = _entityService.rollbackUrn(urn);
+
+      List<AspectRowSummary> deletedRows = result.getRowsRolledBack();
```

Comment:
If we delete an URN, will deletedRows contain multiple versions of the same aspect if they exist?

Reply:
No- a batch delete call just returns the number of rows deleted.  If we delete an urn we dont capture the other rows deleted, just the number.